utils/ramdap.js

```diff
@@ -6,6 +6,9 @@ const B = require('bluebird');
 // Like Ramda.prop except it accepts promises OR values for every argument and _always_ returns a promise
 const propP = curry((prop, obj) => B.all([prop, obj]).then(([prop, obj]) => B.resolve(obj[prop])));
 
+// Like Ramda.props except it accepts promises OR values for every argument and _always_ returns a promise
+const propsP = curry((props, obj) => B.all([props, obj]).then(([props, obj]) => B.all(props.map((prop) => propP(prop, obj)))));
```

Comment:
Since everything is curried, you can do:

```js
const propsP = curry((props, obj) => B.all([props, obj]).then(([props, obj]) => B.all(props.map(propP(__, obj))));
```

...assuming that you import the placeholder from Ramda above.

Reply:
Updated in most recent commit